src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/OpenSslX509ChainProcessor.cs

```diff
@@ -975,10 +975,13 @@ internal int VerifyCallback(int ok, IntPtr ctx)
                             return 0;
                         }
 
-                        // We don't report "OK" as an error.
-                        // For compatibility with Windows / .NET Framework, do not report X509_V_CRL_NOT_YET_VALID.
+                        // * We don't report "OK" as an error.
+                        // * For compatibility with Windows / .NET Framework, do not report X509_V_CRL_NOT_YET_VALID.
+                        // * X509_V_ERR_DIFFERENT_CRL_SCOPE will result in X509_V_ERR_UNABLE_TO_GET_CRL
+                        //   which will trigger OCSP, so is ignorable.
                         if (errorCode != Interop.Crypto.X509VerifyStatusCode.X509_V_OK &&
-                            errorCode != Interop.Crypto.X509VerifyStatusCode.X509_V_ERR_CRL_NOT_YET_VALID)
+                            errorCode != Interop.Crypto.X509VerifyStatusCode.X509_V_ERR_CRL_NOT_YET_VALID &&
```

Comment:
X509_V_ERR_PERMITTED_VIOLATION also seems like something which could be ignored

Reply:
That's "A name constraint violation occurred in the permitted subtrees." which might mean "this certificate has name constraints that exceed the bounds of the parent"... which probably isn't something we'd want to ignore.

I don't really want to ignore things until we can see it, and understand that we're mapping it the same as on Windows :smile:.